api/tests/unit/infrastructure/datasources/airtable/skill-datasource_test.js

```diff
@@ -145,4 +145,53 @@ describe('Unit | Infrastructure | Datasource | Airtable | SkillDatasource', () =
     });
   });
 
+  describe('#findOperativeByCompetenceId', function() {
+
+    beforeEach(() => {
+      const acquix1 = new AirtableRecord('Acquis', 'recAcquix1', {
```

Comment:
on pourrait créer un builder _skillRawAirTableFixture_ [(cf. tubeRawAirTableFixture)](https://github.com/1024pix/pix/blob/PIX-820-keep-out-of-date-skill/api/tests/tooling/fixtures/infrastructure/tubeRawAirTableFixture.js)

Reply:
Le builder existe déjà du coup on l'a utilisé !